test/test-server/fcpm_handlers.go

```diff
@@ -118,3 +118,60 @@ func handleFcpmRegions(t *testing.T) http.HandlerFunc {
 		require.NoError(t, err)
 	}
 }
+
+func handleFcpmIamBindings(t *testing.T) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		var v any
+
+		switch r.Method {
+		case http.MethodGet:
+			usWest1 := flinkv2.FcpmV2IamBinding{
+				Id:           flinkv2.PtrString("fiam-123"),
+				Region:       flinkv2.PtrString("us-west-1"),
+				Cloud:        flinkv2.PtrString("aws"),
+				Environment:  flinkv2.NewGlobalObjectReference("env-123", "", ""),
+				IdentityPool: flinkv2.NewGlobalObjectReference("pool-123", "", ""),
+			}
+			usWest2 := flinkv2.FcpmV2IamBinding{
+				Id:           flinkv2.PtrString("fiam-456"),
+				Region:       flinkv2.PtrString("us-west-2"),
+				Cloud:        flinkv2.PtrString("aws"),
+				Environment:  flinkv2.NewGlobalObjectReference("env-456", "", ""),
+				IdentityPool: flinkv2.NewGlobalObjectReference("pool-456", "", ""),
+			}
+			gcp := flinkv2.FcpmV2IamBinding{
+				Id:           flinkv2.PtrString("fiam-789"),
+				Region:       flinkv2.PtrString("us-central1"),
+				Cloud:        flinkv2.PtrString("gcp"),
+				Environment:  flinkv2.NewGlobalObjectReference("env-789", "", ""),
+				IdentityPool: flinkv2.NewGlobalObjectReference("pool-789", "", ""),
+			}
+
+			iamBindings := []flinkv2.FcpmV2IamBinding{usWest1, usWest2, gcp}
+			if r.URL.Query().Get("cloud") == "aws" {
+				iamBindings = []flinkv2.FcpmV2IamBinding{usWest1, usWest2}
+			}
+			if r.URL.Query().Get("region") == "us-west-1" {
+				iamBindings = []flinkv2.FcpmV2IamBinding{usWest1}
+			}
+			if r.URL.Query().Get("identity_pool") == "pool-123" {
+				iamBindings = []flinkv2.FcpmV2IamBinding{usWest1}
+			}
+			v = flinkv2.FcpmV2IamBindingList{Data: iamBindings}
+		case http.MethodPost:
+			create := new(flinkv2.FcpmV2IamBinding)
+			err := json.NewDecoder(r.Body).Decode(create)
+			require.NoError(t, err)
+
+			create.Id = flinkv2.PtrString("fiam-123")
+			v = create
+		}
+
+		err := json.NewEncoder(w).Encode(v)
+		require.NoError(t, err)
+	}
+}
+
+func handleFcpmIamBindingsId(t *testing.T) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {}
```

Comment:
Any reason this is empty?

Reply:
There's only the DELETE method for the ID route, but I don't want to manipulate the response here. I had to add an empty handler though, because otherwise the tests would fail.
Is there another way to do this?

Reply:
Ah I see, maybe add a comment explaining why it's empty since this an unusual case?

Reply:
Added a comment!